scripts/antlr4/Cypher.g4

```diff
@@ -325,7 +325,7 @@ kU_IfNotExists
     : IF SP NOT SP EXISTS ;
 
 kU_CreateNodeTable
-    : CREATE SP NODE SP TABLE SP (kU_IfNotExists SP)? oC_SchemaName SP? '(' SP? kU_PropertyDefinitions SP? ( ',' SP? kU_CreateNodeConstraint )? SP? ')' ;
+    : CREATE SP NODE SP TABLE SP (kU_IfNotExists SP)? oC_SchemaName ( SP? '(' SP? kU_PropertyDefinitions SP? ( ',' SP? kU_CreateNodeConstraint )? SP? ')' | SP AS SP? '(' SP? oC_Query SP? ')' ) ;
```

Comment:
I suspect `()` is not needed in PG grammar. Can u double check? Let's stick to PG on the syntax for this.

Reply:
Fixed with https://github.com/kuzudb/kuzu/pull/5354/commits/f989e4ff19b26efe614daadde7dfe645ed52ae44